content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -0,0 +1,620 @@
+{{< /tip >}}
+
+{{< text bash >}}
+$ kubectl create ns client-b
+$ kubectl label namespace client-b istio-injection=enabled
+$ kubectl apply -f samples/sleep/sleep.yaml -n client-b
+$ kubectl wait --for condition=available  deployment/sleep -n client-b
+{{< /text >}}
+
+Wait for the pods to get to Running state in the client-b namespace before continuing.
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -n client-b  -- curl httpbin.ambient-demo.svc.cluster.local:8000 -s | grep title -m 1
+    <title>httpbin.org</title>
+{{< /text >}}
+
+Again, we can further verify from viewing the logs of the ztunnel pod (not shown in the example) at the destination node that traffic does in fact use the HBONE and CONNECT based path from the sidecar proxy based source client pod to the ambient based destination service pod. Additionally not shown but we can also verify that unlike the previous subsection, in this case even if we apply a PeerAuthentication resource to the namespace tagged for ambient mode, communication continues between client and service pods since both use the HBONE control and data planes relying on mTLS.
+
+### North-South Ingress Gateway to Ambient backend pods {#nsingress2ambient}
+
+We now describe a use case for North-South traffic with an Istio Gateway exposing the httpbin service via the Kubernetes Gateway API. The gateway itself is running in a non-Ambient namespace and may be an existing gateway that is also exposing other services that are provided by non-ambient pods. Hence this example shows that ambient workloads can also interoperate with Istio gateways that need not themselves be running in namespaces tagged for ambient mode of operation.
+
+For this example, we will use `metallb` to provide a load balancer service on an IP addresses that is reachable from outside the cluster. However the same example also works with other forms of North-South load balancing options. For the example below, we assume that you have already installed `metallb` in this cluster to provided the load balancer service including a pool of IP addresses for `metallb` to use for exposing services externally. Refer to the [`metallb` guide for Kind](https://kind.sigs.k8s.io/docs/user/loadbalancer/) for instructions on setting up `metallb` on Kind clusters or refer to the instructions from the [`metallb` documentation](https://metallb.universe.tf/installation/) appropriate for your environment.
+
+In this example we use the Kubernetes Gateway API for configuring the N-S gateway. Since this API is not currently provided as default in Kubernetes and Kind distributions, we have to install the API CRDs first as shown in the example.
```

Comment:
```suggestion
In this example we use the Kubernetes Gateway API for configuring the N-S gateway. Since this API is not currently provided as default in Kubernetes and kind distributions, we have to install the API CRDs first as shown in the example.
```

Reply:
Done